terraform/aks/workspace_variables/migration_aks.tfvars.json

```diff
@@ -0,0 +1,9 @@
+{
+  "postgres_flexible_server_sku": "GP_Standard_D2ds_v4",
```

Comment:
If it's not for production use, can we use a cheaper one? Like the default in https://github.com/DFE-Digital/terraform-modules/blob/main/aks/postgres/variables.tf#L85

Reply:
Yes, will amend.